internal/controller/controller.go

```diff
@@ -90,17 +95,20 @@ func (c *Controller) Start(ctx context.Context) error {
 	rep, err = reporter.Start(ctx, &reporter.Config{
 		CollAgentAddr:          c.config.CollAgentAddr,
 		DisableTLS:             c.config.DisableTLS,
-		MaxRPCMsgSize:          32 << 20, // 32 MiB
+		MaxRPCMsgSize:          32 * MiB,
 		MaxGRPCRetries:         5,
 		GRPCOperationTimeout:   intervals.GRPCOperationTimeout(),
 		GRPCStartupBackoffTime: intervals.GRPCStartupBackoffTime(),
 		GRPCConnectionTimeout:  intervals.GRPCConnectionTimeout(),
 		ReportInterval:         intervals.ReportInterval(),
-		CacheSize:              traceHandlerCacheSize,
-		SamplesPerSecond:       c.config.SamplesPerSecond,
-		KernelVersion:          kernelVersion,
-		HostName:               hostname,
-		IPAddress:              sourceIP,
+		ExecutablesCacheSize:   4 * KiB,
+		// Next step: Calculate FramesCacheSize from numCores and samplingRate.
+		FramesCacheSize:  64 * KiB,
+		CGroupCacheSize:  1 * KiB,
```

Comment:
The unit for `FramesCacheSize` and `CGroupCacheSize` should not be of type `KiB`. `KiB` suggests a memory limit, while these numbers don't take the size of the element for the respective cache into account.

Reply:
We use the ISO stuff everywhere in `2^N` (1024, 4096 etc). What do you suggest instead? Should we revert to `1<<10` etc? I think that is less readable.

Reply:
Or are you suggesting the config member variables should be renamed to e.g. `FrameCacheMaxItems`?

Reply:
Sorry my bad, KiB really doesn't make any sense here =)